ursa/graph/graph.py

```diff
@@ -1,22 +1,32 @@
 import ray
 from .vertex import _Vertex
 from .vertex import _DeletedVertex
+from .utils import write_vertex
 
 
 @ray.remote(num_cpus=2)
 class Graph(object):
     """This object contains reference and connection information for a graph.
 
     @field vertices: The dictionary of _Vertex objects.
+    @field versions_to_store: The number of versions to keep in memory for each
```

Comment:
Preferably, this data would be stored in the `LocalManager` object. The data can be passed in from the call to `clean_old_rows`. This way, we can update that data without bogging down the data structure holding it.

Reply:
Resolved this as well as graph_id.